variables.tf

```
@@ -4,36 +4,73 @@
+  description = "The description to associate with the IAM role and attached policy that allows full access to the assessment images S3 bucket."
+  default     = "Allows full access to the S3 bucket where assessment images are stored."
 }
 
-variable "aws_availability_zone" {
+variable "assessmentimagesbucketfullaccess_role_name" {
   type        = string
-  description = "The AWS availability zone to deploy into (e.g. a, b, c, etc.)"
-  default     = "a"
+  description = "The name to associate with the IAM role and attached policy that allows full access to the assessment images S3 bucket."
+  default     = "AssessmentImagesBucketFullAccess"
+}
+
+variable "assume_assessmentimagesbucketfullaccess_roles_policy_description" {
+  type        = string
+  description = "The description to associate with the IAM policy that allows a user to assume the IAM roles that allow access to the assessment images S3 bucket in the Images (Production) and Images (Staging) accounts."
+  default     = "Allows assumption of the roles in the Images (Production) and Images (Staging) accounts that allow full access to the assessment images S3 bucket."
 }
 
 variable "aws_region" {
   type        = string
-  description = "The AWS region to deploy into (e.g. us-east-1)"
+  description = "The AWS region to use for the account provisioners (e.g. \"us-east-1\")."
   default     = "us-east-1"
 }
 
+variable "provisionaccount_role_name" {
+  type        = string
+  description = "The name of the IAM role that allows sufficient permissions to provision all AWS resources in the account."
+  default     = "ProvisionAccount"
+}
+
+variable "provisionassessmentimagesbucket_policy_description" {
+  type        = string
+  description = "The description to associate with the IAM policy that allows provisioning of the assessment images S3 Bucket in the Images account."
+  default     = "Allows provisioning of assessment images S3 resources in the Images account."
+}
+
+variable "provisionassessmentimagesbucket_policy_name" {
+  type        = string
+  description = "The name to associate with the IAM policy that allows provisioning of the assessment images S3 Bucket in the Images account."
+  default     = "ProvisionAssessmentImagesBucket"
+}
+
+variable "read_terraform_state_role_name" {
+  type        = string
+  description = "The name to associate with the IAM role and attached policy that allows read-only access to the cool-images-assessment-images state in the S3 bucket where Terraform state is stored."
+  default     = "ReadImagesAssessmentImagesTerraformState"
+}
+
 variable "tags" {
   type        = map(string)
   description = "Tags to apply to all AWS resources created"
   default     = {}
 }
+
+variable "user_name" {
+  type        = string
+  description = "The name of the user to create in the Users account which will have the attached policy to assume roles allowing access to the assessment images buckets in the Images (Production) and Images (Staging) accounts."
```

To me this reads more cleanly, but take it or leave it:
```suggestion
  description = "The name of the user to create in the Users account that can assume roles allowing access to the assessment images buckets in the Images (Production) and Images (Staging) accounts."
```

---

I tweaked this based on your feedback in c02ab6f. Please let me know if you think it need further refinement.

---

Looks great to me. 👍